libs/blocks/aside/aside.css

```
@@ -46,13 +46,34 @@
 }
 
 .aside .split-image img,
-.aside .background img {
+.aside .background img,
+.aside .split-image video,
+.aside .background video {
   object-fit: cover;
   min-height: 700px;
   width: 100%;
   height: 100%;
 }
 
+.aside .background video {
+  display: block;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.aside .background.has-video {
+  bottom: 0;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  right: 0;
+  top: 0;
```

```suggestion
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  overflow: hidden;
```

nit: grouping these makes it more readable

---

@fullcolorcoder I have made the requested changes.